wire-ios-sync-engine/Tests/Source/Integration/IntegrationTest.swift

```diff
@@ -58,6 +58,7 @@ final class MockAuthenticatedSessionFactory: AuthenticatedSessionFactory {
         for account: Account,
         coreDataStack: CoreDataStack,
         configuration: ZMUserSession.Configuration,
+        analyticsSessionConfiguration: AnalyticsSessionConfiguration? = nil,
```

Comment:
I suggest to not provide a default value.
```suggestion
        analyticsSessionConfiguration: AnalyticsSessionConfiguration?,
```